tools/automation/twiml-generator/generator.py

```diff
@@ -24,12 +26,82 @@ def generate_code_sample_filepath(twiml_filepath, language):
     return twiml_filepath.parent.parent / (twiml_filepath.name[:-3] + version + language_spec['extension'])
 
 
+def test(code_generator):
```

Comment:
No I think we need both because they are testing two different scenarios. The generator needs a fast a clear output in case we need to update it in the future. The snippet-tester doesn't cover only the TwiML snippets but all snippets inside api-snippets and makes sure everything runs even if there is a human interaction later on the generated codes.